### ISSUE TITLE
Shitty Vegan info page

### ISSUE THREAD

**Issue by Jamchiller** [oliverjam/shitty-vegan#11]

**Comment by oliverjam** [comment]
I think if we're enforcing account creation we may as well skip a step and make the flow:

shittyvegan.com (not logged in): this explanation page, with a button at the bottom (Log in to get started)
↓ [click log in]
Netlify log in modal
↓ [enter details]
shittyvegan.com (Calendar view)

**Comment by Jamchiller** [comment]
Yep defo, but I think it's important that they can access this info from a logged-in state too, so I could put it together and then we could put it in both the onboarding flow and on the Home screen.

**Comment by oliverjam** [comment]
Yeah good point. I'm not sure I've seen many sites do this well. You either get the marketing site when you're not logged in or the app when you are.

Netlify handles this interestingly: netlify.com is always the marketing site, and always shows a Login link. If you're not logged in the Login link asks for username/pw, then redirects to app.netlify.com. If you're already logged in you still have to click Login, it just redirects you straight away.

<img width="1093" alt="screen shot 2018-08-20 at 16 14 51" src="https://user-images.githubusercontent.com/9408641/44349177-51d79c80-a494-11e8-8b88-7c88aea9a2b9.png">
<img width="1093" alt="screen shot 2018-08-20 at 16 15 05" src="https://user-images.githubusercontent.com/9408641/44349185-53a16000-a494-11e8-958d-d5c28771b37a.png">
